### ISSUE TITLE
loading OSM data from pbf file

### ISSUE THREAD

**Issue by 00krishna** [DeloitteDigitalAPAC/LightOSM.jl#67]
Hello. I was wondering if I can load OSM data from a file--instead of downloading from the internet? I downloaded a big OSM file and I wanted to see if I could import that data into LightOSM for analysis. I did not see this file access stuff in the tutorial or api docs, but I imagine it must be in there somewhere :). Thanks.

**Comment by mmiller-max** [comment]
Hi, thanks for your question!

Currently LightOSM can read from a `.osm` file or a `.json` file, unfortunately `.pbf` is not currently supported - I suggest we leave this issue open to track that and we can try to add it as a feature in future.

[Here](https://deloittedigitalapac.github.io/LightOSM.jl/docs/create_graph/#LightOSM.graph_from_file) is a link to the function to load a graph from a file.

**Comment by 00krishna** [comment]
@mmiller-max Thanks so much for the info. I am not sure if it will help, but in looking at the OpenStreetMapX.jl package, the developers indicated that they can read `pbf` files. i believe that `LightOSM` also drew a lot of inspiration from `OpenStreetMapX.` I wonder if there is a way to leverage what has already been developed, to allow `LightOSM` to pull in `pbf` files?

**Comment by mmiller-max** [comment]
Yes, I'm sure we could do something similar to `OpenStreenMapX.jl`, or similarly [`OpenStreetMapParser.jl`](https://github.com/yeesian/OpenStreetMapParser.jl) or [`OpenStreetMapPBF.jl`](https://github.com/mattwigway/OpenStreetMapPBF.jl). Just needs someone to take a look and see how they've been done and make a PR :)
